src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final String INDEX_EXCEPTION_MESSAGE = "We haven't element at index ";
+    private static final String AVAILABILITY_EXCEPTION_MESSAGE = "We don't have element: ";
```

Comment:
I personally think that it is not necessary here but if you want to use them, it`s your choice :)
Also some minor phrasing mistakes
```suggestion
    private static final String INDEX_EXCEPTION_MESSAGE = "Element missing for given index ";
    private static final String AVAILABILITY_EXCEPTION_MESSAGE = "Element missing: ";
```